specs/integration/events/conversation_message.spec.ts

```diff
@@ -93,39 +138,27 @@ describe('Event conversation:message', () => {
       })
 
       it('returns a 200', async () => {
-        await agent.post('/webhooks/trigger').send(webhookPayload).expect(200)
-
-        // Wait for the worker to pickup the job and process it
-        await waitForEvent('completed', webhookJob.worker, { timeout: 500 });
+        await trigger()
```

Comment:
Was curious about what you think of tests like this, where the expectation is within the function. Seems to introduce a level of indirection to our test and obfuscates the expectations.

Reply:
in the `returns a 200` test, don't use trigger. Use it in all the other tests